testtools/UART_interface/input.sh

```diff
@@ -5,4 +5,4 @@
 set -e
 
 # basic setup of input script for python
```

Comment:
add a comment referring to the "exit" command doing a reset after set_iothub_connection_string. It will help describe why we are doing this

Reply:
I added a complete steps comment.